lib/clashom.gi

```diff
@@ -2843,19 +2859,22 @@ local r,	#radical
     if radsize>1 then
       hom:=ser.factorhom;
       ntrihom:=true;
-      f:=Image(hom);
       # we need centralizers
       #fants:=Filtered(NormalSubgroups(f),x->Size(x)>1 and Size(x)<Size(f));
     else
       if IsPermGroup(G) then
 	hom:=SmallerDegreePermutationRepresentation(G:cheap);
 	ntrihom:=not IsOne(hom);;
+      elif IsPermGroup(Range(ser.factorhom))
+        and not IsPermGroup(Source(ser.factorhom)) then
+	ntrihom:=false;
```

Comment:
Tabs -> spaces?

Reply:
Not deliberately, but maybe a side-effect of my editor. Are you argue I should change *from* tabs (there are many more in the file), or *to* tabs (since there are so many more?